scripts/manual-release.sh

```diff
@@ -40,13 +41,13 @@ else
   EXTRAVERSION_NUMBER=0
 fi
 
-RELEASE="$MAJOR.$MINOR.$PATCH$EXTRAVERSION"
-PRELEASE="$MAJOR.$MINOR.$PATCH.$EXTRAVERSION_NUMBER"
+RELEASE="$MAJOR.$MINOR.$PATCH.$TWEAK$EXTRAVERSION"
+PRELEASE="$MAJOR.$MINOR.$PATCH.$TWEAK"
```

Comment:
This needs to be reverted, as it would break the release process which relies on -rc1, -rc2. 

I don't know what we have used or would use the TWEAK version for, but we should probably just use the PATCH version instead.